conf/application.conf

```diff
@@ -123,7 +123,6 @@ Dev {
       sso {
         host = localhost
         port = 9974
-        protocol = https
```

Comment:
Why was this removed?

Reply:
when running it locally I was getting a no SSL certificate error (or something like this) when this endpoint was called because It should be http protocol in local...? http is the default if no protocol is provided